.travis.yml

```diff
@@ -0,0 +1,37 @@
+# Set up Go
+language: go
+
+go:
+  - "1.10"
+
+go_import_path: github.com/activecm/rita
+
+# Set up Docker
+sudo: required
+
+services:
+  - docker
+
+# Only clone the most recent commit.
+git:
+  depth: 1
+
+# Don't email the test results
+notifications:
+  email: false
+
+before_install:
+  # Get RITA go dependencies
+  - go get ./...
```

Comment:
Is `dep` intentionally not used here?  If so, could you add a comment explaining why? (e.g. "We purposely aren't using vendored dependencies in order to catch any upstream problems early")

Reply:
Added in go dep.